site/sfguides/src/application-control-framework/application-control-framework.md

```diff
@@ -0,0 +1,490 @@
+author: Marc Henderson
+id: application-control-framework
+summary: A guide to building a Native App with consumer controls and limits.
+<!--- Categories below should be hyphenated, i.e., Getting-Started. Do not leave blank. Visit site for available categories. -->
```

Comment:
@sfc-gh-mhenderson Delete this comment.

Reply:
@iamontheinet comment deleted and update pushed